### PR TITLE
Composite databases will not appear in requestedHosting

### PR DESCRIPTION
**Pull request by nick-giles-neo**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-cypher-602.surge.sh

**Comment by nick-giles-neo** [comment]
Related to https://trello.com/c/Tql73nvn/5844-s4pan-prisma-cloud-cider-question-on-show-servers-output-for-composite-databases

**Review by JPryce-Aklundh — APPROVED** [review]
Lgtm 👍

File: modules/ROOT/pages/administration/servers.adoc [hunk 1]
```diff
@@ -237,7 +237,8 @@ The table of results shows information about the servers:
 | {check-mark}
 
 | requestedHosting
-| A list of databases that should be hosted on the server, decided by the allocator.
+| A list of databases that should be hosted on the server.
+Composite databases do not currently appear in this list, though they will appear in `hosting` for all servers.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/administration/servers.adoc]:
```suggestion
Composite databases do not currently appear in this list, though they do appear in `hosting` for all servers.
```

**Pull request merged**